pydundas/rest/constant.py

```diff
@@ -11,11 +11,21 @@ class ConstantNameError(KeyError):
 class Constant:
     """Expose Dundas constants with human names"""
 
-    # In a javascript console:
-    # for (d in dundas.constants) { if (d.endsWith('_ID')){console.log(d + ": " + dundas.constants[d])}}
+    # There is not way to get the constants from the rest API.
+    # To find them I got the list from the documentation page:
+    # https://www.dundas.com/support/api-docs/js/#API%20Reference/dundas/constants.html
+    # And then got the IDs in javascript:
+    # cs = [ADMIN_ACCOUNT_ID", "ALL_TOKEN_DEFINITION_ID", ...];
+    # for (i=0; i<cs.length; i++) {
+    #     console.log("        \"" + cs[i] + "\": \"" + dundas.constants[cs[i]]+ "\",")
```

Comment:
For the sake of easy diffing, maybe add a final `sort`? Looks like data does not always return in the same order.

Reply:
It was badly sorted, now it's OK.